### PR TITLE
Always use latest runtime and cli for sample validation

### PR DESCRIPTION
**Pull request by berndverst**
# Description

Always use latest runtime and cli for sample validation

This change was already deployed in the GO-SDK: https://github.com/dapr/go-sdk/pull/192/
You can see an example run from the past hour here: https://github.com/dapr/go-sdk/runs/3163493512?check_suite_focus=true

### CODE REVIEW

**Comment by berndverst** [comment]
@wcs1only the code coverage is fine .. change is only in:

`.tox/py37/lib/python3.7` which I didn't touch.

https://app.codecov.io/gh/dapr/python-sdk/compare/247/tree/.tox/py37/lib/python3.7

**Comment by wcs1only** [comment]
> @wcs1only the code coverage is fine .. change is only in:
> 
> `.tox/py37/lib/python3.7` which I didn't touch.
> 
> https://app.codecov.io/gh/dapr/python-sdk/compare/247/tree/.tox/py37/lib/python3.7

Yeah, fixed this via: #246

**Comment by berndverst** [comment]
@wcs1only good timing. Looks good now! :)

File: .github/workflows/validate_examples.yaml [hunk 1]
```diff
@@ -15,17 +15,26 @@ jobs:
...
-      DAPR_RUNTIME_VER: 1.2.0-rc.5
       DAPR_INSTALL_URL: https://raw.githubusercontent.com/dapr/cli/master/install/install.sh
       DAPR_CLI_REF:
       DAPR_REF: 
     steps:
       - uses: actions/checkout@v2
+      - uses: azure/setup-helm@v1
+      - name: Determine latest Dapr Runtime version
+        run: |
+          helm repo add dapr https://dapr.github.io/helm-charts/ && helm repo update && export RUNTIME_VERSION=$(helm show chart dapr/dapr --devel | grep "appVersion: [0-9]\+\.[0-9]\+\.[0-9]\+.*" | cut -d ' ' -f2)
```
Comment (wcs1only) [inline .github/workflows/validate_examples.yaml:30]: Helm has a versions command you could use to save you having to parse this yourself:
```suggestion
          helm repo add dapr https://dapr.github.io/helm-charts/ && helm repo update && export RUNTIME_VERSION=$(helm search repo dapr/dapr --devel --versions | awk '/dapr\/dapr/ {print $3; exit}' )
```

**Review by wcs1only — COMMENTED** [review]
Love it, couple of bash nits.

**Review by wcs1only — APPROVED** [review]

**Pull request merged**